core/src/main/java/com/linecorp/armeria/client/retry/Backoff.java

```diff
@@ -84,19 +85,60 @@ static Backoff exponential(long initialDelayMillis, long maxDelayMillis, double
     }
 
     /**
-     * Returns a {@link Backoff} that provides a delay that increases using
+     * Returns a {@link Backoff} that provides a jitter value for the backoff delay using
      * <a href="https://www.awsarchitectureblog.com/2015/03/backoff.html">full jitter</a> strategy.
+     * The jitter value will be multiplied to backoff delay to calculate the bound. For example, if the
+     * calculated delay using {@link Backoff#exponential(long, long)} is 1000 millis and the provided
+     * jitter value is 0.3, the ultimate backoff delay will be chosen between 1000 * (1 - 0.3)
+     * and 1000 * (1 + 0.3) by {@link ThreadLocalRandom}.
+     * The rate value should be between {@code 0.0f} and {@code 1.0f}.
```

Comment:
- The jitter value will be multiplied to backoff delay to calculate the bound. -> Could you rephrase this so it's a little bit easier to understand?
- calculated delay using ... is 1000 millis -> the delay returned by `{@link #nextDelayMillis(...)}` is 1000 milliseconds
- 0.0f, 1.0f -> 0.0 and 1.0 (f is for float and we use double here)